lib/apiservers/portlayer/swagger.json

```diff
@@ -2408,6 +2686,138 @@
 					}
 				}
 			}
+		},
+		"LoggingJoinConfig": {
+			"type": "object",
+			"required": [
+				"handle"
+			],
+			"properties": {
+				"handle": {
+					"type": "object"
```

Comment:
So I get enabling rich handle types. What I don't quite like is passing in `interface{}` to port layer api's. Can we have a base type for handles (or can we assume the handle types share behavior/attributes, like they must be serializable)? Also we are doing `object -> interface{} -> string -> exec.Handle -> interface{} -> exec.Handle` (from http request to port layer api call). That seems a little ridiculous to me. Is `object -> interface{} -> Handle` not possible?